src/BiddingFacilityDriver.cpp

```diff
@@ -12,28 +12,20 @@ int BiddingFacilityDriver(int argc, char *argv[]) {
     // Determine Number Of Players
     std::cout << "Welcome to Eight-Minute-Empire, how many Players will be playing?" << std::endl;
     std::cin >> numOfPlayers;
-
     std::vector<Player*> players;
 
+    // Create all Players in Player vector
     for(int i = 0; i < numOfPlayers; i++) {
         std::cout << "What is your name?" << std::endl;
         std::cin >> name;
-        std::cout << "What is your date of birth? (YYMMDD)" << std::endl;
+        std::cout << "How many years old are you? (eg 18)" << std::endl;
```

Comment:
```suggestion
        std::cout << "How old are you? (eg 18)" << std::endl;
```

So we sound less like a lizard person trying to sound like a human

Reply:
LMAO